chaosreader/komand_chaosreader/actions/run/action.py

```diff
@@ -29,152 +29,118 @@ def run(self, params={}):
         files = []
         sessions = []
 
-        # Set file path to store file
-        dir = "/tmp/chaosreader"
+        temp_dir = f"{tempfile.gettempdir()}/chaosreader"
         file_name = "chaosreader.pcap"
-        full_path = "%s/%s" % (dir, file_name)
+        full_path = f"{temp_dir}/{file_name}"
 
-        # Set variables needed when calling chaosreader
         binary = "/usr/bin/chaosreader"
-        if options:
-          cmd = "%s %s -D %s %s" % (binary, full_path, dir, options)
-        else:
-          cmd = "%s %s -D %s" % (binary, full_path, dir)
+        cmd = f"{binary} {full_path} -D {temp_dir}"
 
-        # Set options to exclude
         if exclude == "Info":
-          options = "-I"
-          self.logger.info('Exluding info files')
+            self.logger.info('Exluding info files')
         elif exclude == "Raw":
-          options = "-R"
-          self.logger.info('Exluding raw files')
+            self.logger.info('Exluding raw files')
         elif exclude == "TCP":
-          options = "-T"
-          self.logger.info('Exluding TCP traffic')
+            self.logger.info('Exluding TCP traffic')
         elif exclude == "UDP":
-          options = "-U"
-          self.logger.info('Exluding UDP traffic')
+            self.logger.info('Exluding UDP traffic')
         elif exclude == "ICMP":
-          options = "-Y"  
-          self.logger.info('Exluding ICMP traffic')
+            self.logger.info('Exluding ICMP traffic')
 
-        # Verify string is base64
```

Comment:
keep comments unless they are no longer valid

Reply:
Every comment, unless they are no longer valid, should be leaved? Even those which are redundant?